include/core/parameters.h

```diff
@@ -1110,19 +1110,16 @@ namespace Parameters
   };
 
   /**
-   * @brief FlowControl - Set volumetric flow rate on a boundary id
-   * toward the normal direction of this wall.
+   * @brief FlowControl - Set average velocity on a boundary (CFD) or the domain
+   * (CFD-DEM).
    */
   struct DynamicFlowControl
   {
-    // Type of verbosity for the flow control
-    Verbosity verbosity;
-
     // Enable flow control
     bool enable_flow_control;
 
-    // Volumetric flow rate (L^3/t)
-    double flow_rate_0;
+    // Average velocity target (L/t)
+    double average_velocity_0;
```

Comment:
Why not call the variable average_velocity_target instead?

Reply:
It's (m_dot/A_c)^0 in the reference, but I changed the equation a lot, so I guess I could you average_velocity_target now